include/topgg/webhook.h

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include <dpp/dpp.h>
+#include <nlohmann/json.hpp>
+#include <functional>
+#include <string>
+
+namespace topgg {
+
+/**
+ * @brief Data received from a bot vote webhook
+ */
+struct bot_vote_data {
+    dpp::snowflake user_id;          ///< ID of the user who voted
```

Comment:
These docstrings should've used the proper format consistent with the other classes in this library (e.g: use `@brief`, `@version`, etc).